ImportedResourcesCleaner.ps1

```diff
@@ -0,0 +1,7 @@
+Get-ChildItem . -recurse -include *.meta | remove-item
```

Comment:
I think that is better if we don't have this first line atm, as we dont want to delete by accident metafiles in the Game project. We can add this after the split of Assets is done in a new repository.